src/scripts/enterprise-configure.sh

```diff
@@ -248,55 +207,88 @@ EOF
      log  "err: creating file ${DATA_GEN_FILE}. you will need to manually configure the metanode"
      exit 1
   fi
+  start_service influxdb 
 }
 datanode_count()
 {
     #checking to see if the $COUNT parameter is set 
   log "[datanode_count] checking COUNT parameter"
 
   if [ -z "${COUNT}" ]; then
-    log "err: please set \$_COUNT parameter..."
+    log "err: datanode count not set - please set -c parameter."
 
     exit 1
   fi
 }
+telegraf()
+{
+  #generate and stage new telegraf configuration file
+  log "[configure_telegraf] generating new telegraf configuration file at ${TELEGRAF_CONFIG_FILE}"
+
+    touch "${TELEGRAF_CONFIG_FILE}"
+    if [ $? -eq 0 ]; then
+      cat > "${TELEGRAF_CONFIG_FILE}" <<-EOF
```

Comment:
I think we actually include a Telegraf config file in the images. Just need to copy the config over ([file](https://github.com/influxdata/azure-resource-manager-influxdb-enterprise/blob/master/packer/config/telegraf-data.conf) and [path](https://github.com/influxdata/azure-resource-manager-influxdb-enterprise/blob/master/packer/scripts/setup-telegraf.sh#L10)).